src/iOS.Core/Controllers/LoginAddViewController.cs

```diff
@@ -121,6 +179,19 @@ public override void ViewDidLoad()
             base.ViewDidLoad();
         }
 
+        private void Type_ValueChanged(object sender, EventArgs e)
```

Comment:
🎨 Rename to something more meaningful of what this method does, like `OrganizationCell_ValueChanged`